File: code/src/test/scala/cern/colt/matrix/tint/impl/SparseCCIntMatrix2DViewTest.scala
package cern.colt.matrix.tint.impl

//remove if not needed
import scala.collection.JavaConversions._

class SparseCCIntMatrix2DViewTest(arg0: String) extends SparseCCIntMatrix2DTest(arg0) {

  protected def createMatrices() {
    A = new SparseCCIntMatrix2D(NCOLUMNS, NROWS).viewDice()
    B = new SparseCCIntMatrix2D(NCOLUMNS, NROWS).viewDice()
    Bt = new SparseCCIntMatrix2D(NROWS, NCOLUMNS).viewDice()
  }
}

File: src/test/java/cernx/colt/matrix/tint/impl/SparseCCIntMatrix2DViewTest.java
package cernx.colt.matrix.tint.impl;

public class SparseCCIntMatrix2DViewTest extends SparseCCIntMatrix2DTest {

    public SparseCCIntMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseCCIntMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new SparseCCIntMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new SparseCCIntMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}
